onnxruntime/python/onnxruntime_pybind_state_common.h

```diff
@@ -64,7 +64,7 @@ struct PyInferenceSession {
 
   void AddCustomOpLibraries(const std::vector<std::shared_ptr<CustomOpLibrary>>& custom_op_libraries) {
     if (!custom_op_libraries.empty()) {
-      custom_op_libraries_.reserve(custom_op_libraries_.size() + custom_op_libraries.size());
+      custom_op_libraries_.reserve(custom_op_libraries.size());
```

Comment:
why this change?

Reply:
it is not needed- minor cleanup from my original change.